sroka/api/google_drive/google_drive_api.py

```diff
@@ -142,18 +149,328 @@ def google_drive_sheets_write(data, spreadsheet_id, sheet_range='Sheet1!A1',
     except HttpError as err:
         print("HTTP error occurred. Error:")
         print(err)
-        return None
+        return False
```

Comment:
Why False? We usually use False, if the "success path" returns True. I nother cases False is usually used to indicate that the data was wrong, not missing, so I'd stick to None, of course it may depend on the context, so I am not saying I am right :D